packages/rancher-sriov/generated-changes/overlay/app-README.md

```diff
@@ -0,0 +1,14 @@
+# Rancher SR-IOV Network Operator
+
+This chart is based on the upstream [k8snetworkplumbingwg/sriov-network-operator](https://github.com/k8snetworkplumbingwg/sriov-network-operator) project. The chart deploys the SR-IOV Operator and its CRDs, which is designed to help the user provision and configure the SR-IOV CNI in a cluster that uses [Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni), to provide high performing extra network interfaces to pods.
```

Comment:
Based on offline conversation, we need to make sure this file calls out that:
1) This chart is expected to be deployed on an RKE2 cluster. It has not been tested or designed for any other cluster type at the moment, although it can be deployed in other clusters with some modifications. (Addresses https://github.com/rancher/charts/pull/1211#issuecomment-853288767)
2) A user who is deploying this chart is expected to be fairly knowledgable about Multus, SR-IOV, and the use case behind this chart as it is only meant for **advanced use cases** where multiple CNIs and high performing network interfaces on pods are required. Users who do not need these features are not advised to install this chart.

cc: @cjellick @cbron

Reply:
In the case of the sriov chart, although we can fix the pod tolerations to make it installed successfully in a rke1 cluster,  I think we will keep it as it is today so the failure can also bring the user’s attention and realize the cluster is not an RKE2 cluster.